src/js/shared_controls.js

```diff
@@ -100,8 +100,17 @@ $("input:radio[name='format']").change(function () {
 	$(".format-specific").not("." + gameType.toLowerCase()).hide();
 });
 
+var defaultLevel = 100;
+$("input:radio[name='defaultLevel']").change(function () {
+	defaultLevel = $("input:radio[name='defaultLevel']:checked").val();
+	$("#levelL1").val(defaultLevel);
+	$("#levelR1").val(defaultLevel);
+	$("#levelL1").trigger("change");
+	$("#levelR1").trigger("change");
+});
+
 // auto-calc stats and current HP on change
-$(".level").keyup(function () {
+$(".level").bind("keyup change", function () {
```

Comment:
Why? What's the difference?

Reply:
I can explain this, the purpose is to have both active sets’ stats change automatically when the default level is changed, in accordance with that level. The `keyup` event is not triggered when the level values are changed programmatically, hence the need of the addition of the `change` event, so that the stats and the damage output are recalculated.

Reply:
Oh I see, that makes sense.